flink-connector-oceanbase-cdc/src/main/java/com/ververica/cdc/connectors/oceanbase/table/OceanBaseTableSourceFactory.java

```diff
@@ -163,6 +172,8 @@ public DynamicTableSource createDynamicTableSource(Context context) {
         String tenantName = config.get(TENANT_NAME);
         String databaseName = config.get(DATABASE_NAME);
         String tableName = config.get(TABLE_NAME);
+        String tableList = config.get(TABLE_LIST);
```

Comment:
we can add validation for this three options like  (1) `tableName ` is required when `databaseName` is configured, (2) `tableList` is required when `databaseName` and `tableName` are not configured
https://github.com/ververica/flink-cdc-connectors/blob/7de1587ca83fa98a0ab6cbc54b13c80f620e7f51/flink-connector-oracle-cdc/src/main/java/com/ververica/cdc/connectors/oracle/OracleSource.java#L190

Reply:
Done.